nevergrad/optimization/mutations.py

```diff
@@ -52,6 +52,8 @@ def portfolio_discrete_mutation(self, parent: ArrayLike, u: Optional[int] = None
         dimension = len(parent)
         if u is None:
             u = 1 if dimension == 1 else int(self.random_state.randint(1, dimension))
+        if dimension == 1:  # Dimension 1 is not very important but let us take care of this corner case.
+            return [self.random_state.normal(0., 1.) for _ in parent]
```

Comment:
it's always better to use pure array: `self.random_state.normal(0, 1, size=1)` is enough